graphs/energy/nodes/energy/energy_heat_solar_thermal.converter.ad

```diff
@@ -26,6 +26,7 @@
 - construction_time = 2.0
 - technical_lifetime = 25.0
 - wacc = 0.04
+- sustainability_share = 1.0
```

Comment:
@antw, thanks for setting up this PR! Just a quick question: I noticed that a `sustainability_share` was added to the `energy_heat_solar_thermal` node but not to the `households_final_demand_solar_thermal` and `buildings_final_demand_solar_thermal` nodes. The latter are also in the `primary_energy_demand` group. Is there a reason why these didn't get a `sustainability_share`?

Reply:
> The latter are also in the primary_energy_demand group. Is there a reason why these didn't get a sustainability_share?

The heuristic I used for adding a `sustainability_share` was to find primary_energy_demand nodes where the following two conditions were true:

* The node has no `sustainability_share` already.
* The node's output carriers don't define a `sustainable` value.

When both are true, ETEngine cannot calculate a `sustainability_share` for the node.

In the case of "households_final_demand_solar_thermal" and "buildings_final_demand_solar_thermal", both have solar_thermal as their output carrier. This carrier [defines a value for `sustainable`](https://github.com/quintel/etsource/blob/d77f954a6abf6323a460c512b85c9ca221da2993/carriers/solar_thermal.ad#L1) from which ETEngine can calculate a `sustainability_share` for the nodes.

"energy_heat_solar_thermal", on the other hand, has steam_hot_water as its main output carrier. This carrier does not define a `sustainable` value.

If you don't like the inconsistency, an alternative might be to remove all three nodes from the primary_energy_demand group, and replace them with "[energy_production_solar_thermal](https://engine.energytransitionmodel.com/inspect/858990/graphs/energy/nodes/households_final_demand_solar_thermal)". There are no conversion losses between this node and its outputs so the results should be identical, with the bonus that ETEngine can calculate the `sustainbility_share` from its solar_thermal output.

Reply:
I think that won't work because of the losses on this node:
<img width="737" alt="image" src="https://user-images.githubusercontent.com/32056448/130221210-3276e982-5866-4ce6-9a15-86b98f0a4966.png">

Ideally we should revise the households/buildings implementation of solar thermal at some point so that it is more in line with the district heating node. But I guess that's something for the long list. I'm happy with your current implementation of `sustainability_share` and the two conditions you describe ✌️